Makefile.dep

```diff
@@ -1059,6 +1064,11 @@ FEATURES_OPTIONAL += periph_pm
 # always select provided architecture features
 FEATURES_REQUIRED += $(filter arch_%,$(FEATURES_PROVIDED))
 
+ifneq (,$(filter auto_init stdio_cdc_acm,$(DISABLE_MODULE)))
```

Comment:
```suggestion
ifneq (,$(filter auto_init_usbus stdio_cdc_acm,$(DISABLE_MODULE)))
```

Reply:
Doing this breaks the build with unittests.

Reply:
The blacklist the feature in `unitests`